src/main/java/uk/gov/justice/digital/delius/config/SwaggerConfig.java

```diff
@@ -44,13 +44,11 @@ public Docket offenderApi() {
                         regex("(\\/health.*)"),
                         regex("(\\/api/.*)"),
                         regex("(\\/secure/.*)"))))
-                .build();
-
-        docket.genericModelSubstitutes(Optional.class);
-        docket.directModelSubstitute(ZonedDateTime.class, java.util.Date.class);
-        docket.directModelSubstitute(LocalDateTime.class, java.util.Date.class);
-
-        return docket;
+                .build()
+                .genericModelSubstitutes(Optional.class)
+                .directModelSubstitute(ZonedDateTime.class, java.util.Date.class)
+                .directModelSubstitute(LocalDateTime.class, java.util.Date.class)
+                .forCodeGeneration(true);
```

Comment:
we were getting `Page<KeyValue>` coming out, which swagger treats as invalid.  This fixes that issue - see https://codesoapbox.dev/spring-boot-swagger-fixing-code-generation-issues/